### PR TITLE
Getting rid of output activation

### PR DESCRIPTION
**Pull request by DanielYakubov**
Realizing LeakyReLU was truly a strange choice as an output layer activation function, removing it entirely. Since we ultimately argmax, it should have no real effect. This just makes more sense.

### CODE REVIEW

**Comment by DanielYakubov** [comment]
Please treat this as a draft PR for now. Looks like something went wrong

**Comment by kylebgorman** [comment]
I'm fine with using the fused log softmax in the cross entropy loss as discussed offline. (Worth documenting very briefly.)

Change detector tests need updated, presumably, and it would be nice to run a sweep on English (and Greek and>) Russian to make sure this isn't an obvious negative. The configs in examples/wandb_sweeps are fine for this.

**Comment by kylebgorman** [comment]
nice tests passed

**Comment by DanielYakubov** [comment]
@kylebgorman did you notice it's more accurate in the tests?

I am at an impasse at running the full sweeps. I wanted to, but using the lab computers is truly unbearable. I'm going to do one more greek sanity check

**Comment by kylebgorman** [comment]
> On Mar 28, 2025, at 1:38 PM, Daniel Yakubov ***@***.***> wrote:
> 
> @kylebgorman did you notice it's more accurate in the tests?

YEAH I did 

> I am at an impasse about running the full sweeps. I wanted to, but using the lab computers is truly unbearable. I'm going to do one more greek sanity check

K
